notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        read_instances_uri=READ_INSTANCES_URI
</pre><p>This is the only time <code>READ_INSTANCES_URI</code>  being used and has been defined way earlier. It can be quite confusing for readers. </p><p> </p><p>Something like this maybe more intuitive for readers</p><p><code>bq_read_instances_uri = create_bq_table(bq_query=read_instances_query)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>